src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -115,6 +115,12 @@ public static bool AzureFilesAppSettingsExist(this IEnvironment environment)
                    !string.IsNullOrEmpty(environment.GetEnvironmentVariable(AzureFilesContentShare));
         }
 
+        public static bool IsDotNetInProcSupported(this IEnvironment environment)
+        {
+            return !IsFlexConsumptionSku(environment)
```

Comment:
add unit tests?

Reply:
Validation was performed on this. I don't expect we'd have this logic here for longer than a release, but I can follow up with some additional tests.

Reply:
Yeah, would be good to add tests for these new helpers.